src/app/course/goal-create/goal-creation-page.component.ts

```diff
@@ -9,20 +9,14 @@ import {CategoryService} from "@app/_services/api/category.service"
 import {Category} from "@app/_models"
 import {DifficultyService} from "@app/problems/_services/difficulty.service"
 import {Difficulty} from "@app/_models/difficulty"
+import {Goal, GoalItem} from "@app/_models/goal/goal"
 
 @Component({
     selector: 'app-goal-create',
-    templateUrl: './goal-create.component.html',
-    styleUrls: ['./goal-create.component.scss']
+    templateUrl: './goal-creation-page.component.html',
```

Comment:
Please keep your component in the goal folder as all the related components to the goal page will be there.
The html file name should also be the same format as the component name (i.e "goal-create.component.html")
Same with the scss file.